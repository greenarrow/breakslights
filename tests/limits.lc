MA03
MR03
R00 N00
R01 N01
R02 N02
A00 BCff FC06 OD01 ON02 OX08 S01
A01 BCff FD01 FN02 FX08 S02 FB1
A02 BCff FC04 TD01 TN02 TX04 S02 I1 TB1
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
