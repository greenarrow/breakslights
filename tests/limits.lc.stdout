00 000000 000000 000000 ffffff ffffff ffffff ffffff ffffff ffffff 000000 000000 000000
01 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000
02 000000 000000 000000 ffffff ffffff 000000 000000 ffffff ffffff 000000 000000 000000
00 000000 000000 000000 000000 ffffff ffffff ffffff ffffff ffffff ffffff 000000 000000
01 ffffff ffffff 000000 000000 000000 000000 ffffff ffffff 000000 000000 000000 000000
02 000000 000000 000000 000000 ffffff ffffff ffffff ffffff 000000 000000 000000 000000
00 000000 000000 000000 000000 000000 ffffff ffffff ffffff ffffff ffffff ffffff 000000
01 ffffff ffffff 000000 000000 000000 000000 ffffff ffffff 000000 000000 000000 000000
02 000000 000000 000000 ffffff ffffff 000000 000000 ffffff ffffff 000000 000000 000000
00 000000 000000 000000 000000 000000 000000 ffffff ffffff ffffff ffffff ffffff ffffff
01 ffffff ffffff ffffff 000000 000000 000000 ffffff ffffff ffffff 000000 000000 000000
02 000000 000000 ffffff ffffff 000000 000000 000000 000000 ffffff ffffff 000000 000000
00 ffffff 000000 000000 000000 000000 000000 000000 ffffff ffffff ffffff ffffff ffffff
01 ffffff ffffff ffffff 000000 000000 000000 ffffff ffffff ffffff 000000 000000 000000
02 000000 000000 000000 ffffff ffffff 000000 000000 ffffff ffffff 000000 000000 000000
00 ffffff ffffff 000000 000000 000000 000000 000000 000000 ffffff ffffff ffffff ffffff
01 ffffff ffffff ffffff ffffff 000000 000000 ffffff ffffff ffffff ffffff 000000 000000
02 000000 000000 000000 000000 ffffff ffffff ffffff ffffff 000000 000000 000000 000000
00 000000 000000 ffffff ffffff ffffff ffffff ffffff ffffff 000000 000000 000000 000000
01 ffffff ffffff ffffff ffffff 000000 000000 ffffff ffffff ffffff ffffff 000000 000000
02 000000 000000 000000 ffffff ffffff 000000 000000 ffffff ffffff 000000 000000 000000
00 000000 000000 000000 ffffff ffffff ffffff ffffff ffffff ffffff 000000 000000 000000
01 ffffff ffffff ffffff 000000 000000 000000 ffffff ffffff ffffff 000000 000000 000000
02 000000 000000 ffffff ffffff 000000 000000 000000 000000 ffffff ffffff 000000 000000
00 000000 000000 000000 000000 ffffff ffffff ffffff ffffff ffffff ffffff 000000 000000
01 ffffff ffffff ffffff 000000 000000 000000 ffffff ffffff ffffff 000000 000000 000000
02 000000 000000 000000 ffffff ffffff 000000 000000 ffffff ffffff 000000 000000 000000
00 000000 000000 000000 000000 000000 ffffff ffffff ffffff ffffff ffffff ffffff 000000
01 ffffff ffffff 000000 000000 000000 000000 ffffff ffffff 000000 000000 000000 000000
02 000000 000000 000000 000000 ffffff ffffff ffffff ffffff 000000 000000 000000 000000
00 000000 000000 000000 000000 000000 000000 ffffff ffffff ffffff ffffff ffffff ffffff
01 ffffff ffffff 000000 000000 000000 000000 ffffff ffffff 000000 000000 000000 000000
02 000000 000000 000000 ffffff ffffff 000000 000000 ffffff ffffff 000000 000000 000000
00 ffffff 000000 000000 000000 000000 000000 000000 ffffff ffffff ffffff ffffff ffffff
01 ffffff 000000 000000 000000 000000 000000 ffffff 000000 000000 000000 000000 000000
02 000000 000000 ffffff ffffff 000000 000000 000000 000000 ffffff ffffff 000000 000000
00 ffffff ffffff 000000 000000 000000 000000 000000 000000 ffffff ffffff ffffff ffffff
01 ffffff 000000 000000 000000 000000 000000 ffffff 000000 000000 000000 000000 000000
02 000000 000000 000000 ffffff ffffff 000000 000000 ffffff ffffff 000000 000000 000000
00 000000 000000 ffffff ffffff ffffff ffffff ffffff ffffff 000000 000000 000000 000000
01 ffffff ffffff 000000 000000 000000 000000 ffffff ffffff 000000 000000 000000 000000
02 000000 000000 000000 000000 ffffff ffffff ffffff ffffff 000000 000000 000000 000000
00 000000 000000 000000 ffffff ffffff ffffff ffffff ffffff ffffff 000000 000000 000000
01 ffffff ffffff 000000 000000 000000 000000 ffffff ffffff 000000 000000 000000 000000
02 000000 000000 000000 ffffff ffffff 000000 000000 ffffff ffffff 000000 000000 000000
00 000000 000000 000000 000000 ffffff ffffff ffffff ffffff ffffff ffffff 000000 000000
01 ffffff ffffff ffffff 000000 000000 000000 ffffff ffffff ffffff 000000 000000 000000
02 000000 000000 ffffff ffffff 000000 000000 000000 000000 ffffff ffffff 000000 000000
00 000000 000000 000000 000000 000000 ffffff ffffff ffffff ffffff ffffff ffffff 000000
01 ffffff ffffff ffffff 000000 000000 000000 ffffff ffffff ffffff 000000 000000 000000
02 000000 000000 000000 ffffff ffffff 000000 000000 ffffff ffffff 000000 000000 000000
00 000000 000000 000000 000000 000000 000000 ffffff ffffff ffffff ffffff ffffff ffffff
01 ffffff ffffff ffffff ffffff 000000 000000 ffffff ffffff ffffff ffffff 000000 000000
02 000000 000000 000000 000000 ffffff ffffff ffffff ffffff 000000 000000 000000 000000
00 ffffff 000000 000000 000000 000000 000000 000000 ffffff ffffff ffffff ffffff ffffff
01 ffffff ffffff ffffff ffffff 000000 000000 ffffff ffffff ffffff ffffff 000000 000000
02 000000 000000 000000 ffffff ffffff 000000 000000 ffffff ffffff 000000 000000 000000
00 ffffff ffffff 000000 000000 000000 000000 000000 000000 ffffff ffffff ffffff ffffff
01 ffffff ffffff ffffff 000000 000000 000000 ffffff ffffff ffffff 000000 000000 000000
02 000000 000000 ffffff ffffff 000000 000000 000000 000000 ffffff ffffff 000000 000000
00 000000 000000 ffffff ffffff ffffff ffffff ffffff ffffff 000000 000000 000000 000000
01 ffffff ffffff ffffff 000000 000000 000000 ffffff ffffff ffffff 000000 000000 000000
02 000000 000000 000000 ffffff ffffff 000000 000000 ffffff ffffff 000000 000000 000000
00 000000 000000 000000 ffffff ffffff ffffff ffffff ffffff ffffff 000000 000000 000000
01 ffffff ffffff 000000 000000 000000 000000 ffffff ffffff 000000 000000 000000 000000
02 000000 000000 000000 000000 ffffff ffffff ffffff ffffff 000000 000000 000000 000000
00 000000 000000 000000 000000 ffffff ffffff ffffff ffffff ffffff ffffff 000000 000000
01 ffffff ffffff 000000 000000 000000 000000 ffffff ffffff 000000 000000 000000 000000
02 000000 000000 000000 ffffff ffffff 000000 000000 ffffff ffffff 000000 000000 000000
00 000000 000000 000000 000000 000000 ffffff ffffff ffffff ffffff ffffff ffffff 000000
01 ffffff 000000 000000 000000 000000 000000 ffffff 000000 000000 000000 000000 000000
02 000000 000000 ffffff ffffff 000000 000000 000000 000000 ffffff ffffff 000000 000000
